examples/options/include/traccc/throw_exception.hpp

```diff
@@ -0,0 +1,43 @@
+/** TRACCC library, part of the ACTS project (R&D line)
+ *
+ * (c) 2022 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
+
+// Boost
+#include <boost/program_options.hpp>
+
+namespace po = boost::program_options;
+
+namespace traccc {
+
+enum exception : int {
+    print_help = 0,
+    input_error = 1,
+    no_exception = 2,
+};
+
+int throw_exception(const po::options_description& desc,
```

Comment:
I'm not sure this function (and file) have a good name. Maybe `handle_argument_errors`?

Reply:
Makes sense